docker/conda/environments/cuda11.8_dev.yml

```diff
@@ -111,3 +111,4 @@ dependencies:
         - databricks-connect
         - pytest-kafka==0.6.0
         - pymilvus==2.3.1
+        - milvus
```

Comment:
Lets see if there is any way to use conda for these dependencies.

Reply:
It is out of date https://anaconda.org/conda-forge/pymilvus
`Last upload: 2 years and 4 months ago`